### PR TITLE
Descriptions and diagrams for several standard generator regulations

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes [106](https://github.com/CRESYM/colib0.github.io/issues/106)

### CODE REVIEW

File: pages/models/regulations/AC1C/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Model inputs and output
+
+The input variables are the measured and reference stator voltages, the rotor current and possibly the output voltages of the power system stabilizer, the underexcitation limiter, the overexcitation limiter and the stator current underexcitation and overexcitation limiters.
+
+The output signal is the excitation voltage EfdPu.
+
+## Model parameters
+
+AEx : Gain of saturation function
```
Comment (matoubongrain) [inline pages/models/regulations/AC1C/index.md]: can you put units? if none, put pu

File: pages/models/regulations/AC1C/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Kf : Exciter rate feedback gain
+PositionOel : Input location : (0) none, (1) voltage error summation, (2) take-over at AVR output
+PositionScl : Input location : (0) none, (1) voltage error summation, (2) take-over at AVR output
+PositionUel : Input location : (0) none, (1) voltage error summation, (2) take-over at AVR output
+tA : Voltage regulator time constant in s
+tB : Voltage regulator lag time constant in s
+tC : Voltage regulator lead time constant in s
+tE : Exciter field time constant in s
+tF : Exciter rate feedback time constant in s
+TolLi : Tolerance on limit crossing as a fraction of the difference between initial limits of limited integrator
```
Comment (matoubongrain) [inline pages/models/regulations/AC1C/index.md]: units

File: pages/models/regulations/AC1C/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+layout: page
+title: Standard voltage regulator model
+tags: ["Voltage regulator", "AC1C", "generic"]
+---
+# AC1C model
+
+## Context
+
+This voltage regulator model has been developed by RTE.
+
```
Comment (matoubongrain) [inline pages/models/regulations/AC1C/index.md:14]: can you add material in the ## Model use, assumptions, validity domain and limitations section?

File: pages/models/regulations/AC1C/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+## Model inputs and output
+
+The input variables are the measured and reference stator voltages, the rotor current and possibly the output voltages of the power system stabilizer, the underexcitation limiter, the overexcitation limiter and the stator current underexcitation and overexcitation limiters.
```
Comment (matoubongrain) [inline pages/models/regulations/AC1C/index.md:21]: can you put the name of input variables?

File: pages/models/regulations/AC1C/index.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+The output signal is the excitation voltage EfdPu.
+
```
Comment (matoubongrain) [inline pages/models/regulations/AC1C/index.md:24]: in the diagram, can you remove the unused input variables?

File: pages/models/regulations/AC1C/index.md (hunk 1 above)
Comment (ErwanGuichard49) [reply pages/models/regulations/AC1C/index.md]: I have added 'in pu' without a base.

File: pages/models/regulations/AC1C/index.md (hunk 3 above)
Comment (ErwanGuichard49) [reply pages/models/regulations/AC1C/index.md:14]: I have left this part empty.

File: pages/models/regulations/AC1C/index.md (hunk 4 above)
Comment (ErwanGuichard49) [reply pages/models/regulations/AC1C/index.md]: Done.

File: pages/models/regulations/AC1C/AC1C.drawio.svg
Comment (ErwanGuichard49) [inline pages/models/regulations/AC1C/AC1C.drawio.svg:1]: Why have you modified the limits ? The diagram that I have drawn was accurate.

Comment (ErwanGuichard49) [reply pages/models/regulations/AC1C/AC1C.drawio.svg:1]: I have restored it.

Comment (matoubongrain) [reply pages/models/regulations/AC1C/AC1C.drawio.svg:1]: I have changed it as I have found this diagram in the IEEE standard:
image

**Review by matoubongrain — APPROVED** [review]

**Pull request merged**